### ISSUE TITLE
Regression: Removing an item from reading list opens up an unstyled dialog box

### ISSUE THREAD

**Issue by tabshaikh** [internetarchive/openlibrary#1559]
When removing a book from a reading list this weird dialog box pops up with the broken cross symbol and the background is also broken
![remove](https://user-images.githubusercontent.com/31198893/48315719-d18e8680-e5ff-11e8-89fa-9333d17aa637.png)

**Comment by tabshaikh** [comment]
Hopefully it functions well :)

**Comment by iem5** [comment]
I will try to fix this over the next week or so!

**Comment by LeadSongDog** [comment]
Perhaps  openlibrary/openlibrary/templates/lists/widget.html is what you seek?

**Comment by jdlrobson** [comment]
Hey @erisugita did you get what you needed here? I believe @LeadSongDog is right. Sorry for the delay (I just got back from vacation)

As well as that, you'll want to look at the CSS in static/css/legacy.less (preferably pulling out the relevant bits into a documented file in static/css/components/

End result should look something like this:

![screen shot 2018-12-12 at 6 32 42 pm](https://user-images.githubusercontent.com/148752/49911739-5b6a8300-fe3c-11e8-8796-7414311e37ba.png)

I'd suggest focusing on the bug fix first by editing the templates and legacy.css (the components stuff we can work on later!)

**Comment by ayushshivani** [comment]
I would like to work on this.

**Comment by tabshaikh** [comment]
Ya sure, go ahead :)

**Comment by ayushshivani** [comment]
The cross icon should displace or not on hovering?

**Comment by jdlrobson** [comment]
legacy.less is not loaded on this page.
This looks like the [cbox component](https://github.com/internetarchive/openlibrary/blob/master/static/css/components/cbox.less)

**Comment by jdlrobson** [comment]
@ayushshivani  you can avoid restarting server by running make css inside bash
E.g.
docker-compose exec bash

**Comment by ayushshivani** [comment]
But in page-user.less from where the page is rendering is importing legacy.less from static/css.And making changes to lagacy.less is giving the desired output.
![screenshot from 2019-01-08 19-50-53](https://user-images.githubusercontent.com/32260760/50836328-e3b34980-1350-11e9-9a4f-4c0ad7da473c.png)

**Comment by LeadSongDog** [comment]
The same oddly rendered closing X box is seen at present on production in the author merge "Are you sure" popup.

**Comment by ayushshivani** [comment]
Can you please tell from where cbox.less is refrenced from? I cannot understand the names written in them.

**Comment by jdlrobson** [comment]
Inside  static/css/js-all.less on L14 there is the rule:

@import (less) 'components/cbox.less';

You can edit components/cbox.less and that should be loaded via JavaScript on page load. Hope that helps!

**Comment by ayushshivani** [comment]
@jdlrobson Thanks for the help !. I am facing one more problem I am not able to find from where the work "Close" ie written just below the "X" is coming from. Can you please provide some help

**Comment by jdlrobson** [comment]
Hey @ayushshivani  I think I'm seeing where all this confusion is coming from.

When I try and remove the book from a list on https://openlibrary.org/books/OL7885696M/Charlotte's_Web
I see:
![screen shot 2019-01-11 at 11 49 38 am](https://user-images.githubusercontent.com/148752/51056190-08006780-1597-11e9-9fa4-32d80e077fe1.png)

This is coming from jquery ui.

So I see exactly why you were using legacy.less now. Sorry about that confusion. I'll push a change today which will make this a lot easier for you. 

I'm sorry for adding to the confusion. Will post later today!

**Comment by jdlrobson** [comment]
Hey @ayushshivani so when I looked into it, I realised this wasn't the best bug to start you off with for our frontend codebase which is in a very transient state. I'm sorry about that!

If you're interested and know a little JavaScript, https://github.com/internetarchive/openlibrary/issues/1588 would be a good task related to this one.

Alternatively https://github.com/internetarchive/openlibrary/issues/1339 has quite a lot of detail and should hopefully be easier to work on.

Sorry again!
